api/src/resource-extraction.js

```diff
@@ -10,6 +10,10 @@ const db = require('./db');
 const environment = require('./environment');
 const logger = require('./logger');
 
+const DDOC_ID = '_design/medic';
```

Comment:
Let's call this `MAIN_DDOC_ID` or something to differentiate it from the ADMIN_DDOC_ID below.